receiver/hostmetricsreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,23 @@
+receivers:
+  hostmetrics:
+    scrapers:
+      cpu:
+  hostmetrics/customname:
+    scrape_interval: 10s
```

Comment:
@tigrannajaryan @bogdandrutu we've been using `collection_interval` in new receivers. We'd like to make them consistent so need to settle on a name. Looks like vmmetrics did use `scrape_interval` previously. I don't have any hard opinions as long as it's consistent. Thoughts?

Reply:
I believe the `vmmetrics` receiver is going to be removed at some point. I'll make it `collection_interval` for now to be consistent with your code, but let me know if this should change.